index.html

```diff
@@ -0,0 +1,131 @@
+     rel="stylesheet">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Bilbo+Swash+Caps&family=Crete+Round&family=El+Messiri&family=Poppins:wght@100;200;400&family=Roboto:wght@100;700&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Bilbo+Swash+Caps&family=Crete+Round&family=El+Messiri&family=Inter&family=Poppins:wght@100&family=Roboto:wght@100;700&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <link
+      rel="stylesheet"
+      href="https://maxcdn.bootstrapcdn.com/font-awesome/4.5.0/css/font-awesome.min.css"
+    />
+    <link href="https://fonts.googleapis.com/css2?family=Bilbo+Swash+Caps&family=Crete+Round&family=El+Messiri&family=Inter&family=Poppins:wght@100&family=Roboto:wght@100;700&display=swap" 
+    rel="stylesheet">
+    <title>Riyaz's Portfolio</title>
+  </head>
+  <body>
+    <div class="container">
+      <header class="site-header">
+        <div id="title-text"><h3>
+          <a class="logo-text" href="#">Riyaz Ahmed</a>
+        </h3>
+      </div>
+        <nav id="navbar">
+          <a class="nav-links" href="#">Portfolio</a>
+          <a class="nav-links" href="#">About</a>
+          <a class="nav-links" href="#">Contact</a>
+          <a><i id="menu" class="material-icons">mail</i></a>
+        </nav>
+        <a href="javascript:void(0);" class="icon">
+          <i class="fa fa-bars"></i>
+        </a>
+      </header>
+
+      <section id="home-content">
+        <div class="left-content">
+            <ul class="desktop-version-list">
+            <li>
+              <a class="social-link" href="#">
+              <img class="img-icons" src="images/github.svg" alt="github" />
+              </a>
+            </li>
+            <li>
+              <a class="social-link" href="#">
+              <img class="img-icons" src="images/linkedin.svg" alt="linkedin" />
+              </a>
+            </li>
+            <li>
+              <a class="social-link" href="#">
+              <img class="img-icons" src="images/angelist.svg" alt="angelist" />
+              </a>
+            </li>
+            <li>
+              <a class="social-link" href="#">
+              <img class="img-icons" src="images/twitter.svg" alt="twitter" />
+              </a>
+            </li>
+            <li>
+              <a class="social-link" href="#">
+              <img class="img-icons" src="images/medium.svg" alt="medium" />
+              </a>
+            </li>
+          </ul>
+        </div>
+        <div class="main-content">
+          <h1 class="name-text">Hey There. <br>
+             I’m Riyaz</h1>
+          <span class="job-title">I’am a Software Developer</span>
```

Comment:
[OPTIONAL] I believe this is a subtitle therefore, according to the evaluation guidelines, this content should be wrapped in a `<hx>` tag.

<img width="572" alt="Screen Shot 2022-05-26 at 8 36 57" src="https://user-images.githubusercontent.com/82426682/170498690-d09b95e2-e6e3-4485-aa0f-a12d2157e98b.png">

However, you may disagree with me, what do you think, is this a subtitle?

Reply:
Not needed as such